Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -481,7 +480,11 @@ BTDiagnostics::DefineCellCenteredMultiFab(int lev)
     ba.coarsen(m_crse_ratio);
     amrex::DistributionMapping dmap = warpx.DistributionMap(lev);
     int ngrow = 1;
+#ifdef WARPX_DIM_RZ
+    int ncomps = (2*WarpX::n_rz_azimuthal_modes - 1) * static_cast<int>(m_cellcenter_varnames.size());
```

Comment:
If I'm not mistaken, this could be also written as
```suggestion
    int ncomps = WarpX::ncomps * static_cast<int>(m_cellcenter_varnames.size());
```